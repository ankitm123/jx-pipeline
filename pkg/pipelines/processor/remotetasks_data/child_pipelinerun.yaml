apiVersion: tekton.dev/v1
kind: PipelineRun
metadata:
  creationTimestamp: null
  name: pullrequest
spec:
  pipelineSpec:
    tasks:
      - name: from-build-pack
        computeResources: {}
        taskSpec:
          metadata: {}
          stepTemplate:
            image: uses:jenkins-x/jx3-pipeline-catalog/tasks/go/pullrequest.yaml@versionStream
            name: ""
            computeResources:
              # override limits for all containers here
              limits: {}
            workingDir: /workspace/source
          steps:
            - image: uses:jenkins-x/jx3-pipeline-catalog/tasks/git-clone/git-clone-pr.yaml@versionStream
              name: ""
              computeResources: {}
            - name: jx-variables
              computeResources:
                # override requests for the pod here
                requests:
                  cpu: 400m
                  memory: 600Mi
            - name: hello-world
              image: golang:1.19
              script: |
                  #!/usr/bin/env sh
                  echo "hello world"
            - name: build-make-linux
              computeResources: {}
            - name: check-registry
              computeResources: {}
            - name: build-container-build
              computeResources: {}
            - name: promote-jx-preview
              computeResources: {}
  taskRunTemplate:
    serviceAccountName: tekton-bot
  timeouts:
    pipeline: 1h0m0s
status: {}
