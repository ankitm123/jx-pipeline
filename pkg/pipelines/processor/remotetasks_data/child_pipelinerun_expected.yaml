apiVersion: tekton.dev/v1
kind: PipelineRun
metadata:
  creationTimestamp: null
  name: pullrequest
spec:
  pipelineSpec:
    tasks:
    - name: git-clone-pr
      taskRef:
        params:
        - name: url
          value: https://github.com/jenkins-x/jx3-pipeline-catalog.git
        - name: revision
          value: versionStream
        - name: pathInRepo
          value: tasks/git-clone/git-clone-pr.yaml
        resolver: git
      workspaces:
      - name: output
        workspace: pipeline-ws
    - name: jx-variables
      runAfter:
      - git-clone-pr
      taskRef:
        params:
        - name: url
          value: https://github.com/jenkins-x/jx3-pipeline-catalog.git
        - name: revision
          value: versionStream
        - name: pathInRepo
          value: tasks/go/pullrequest/jx-variables.yaml
        resolver: git
      workspaces:
      - name: output
        workspace: pipeline-ws
    - name: hello-world
      runAfter:
      - jx-variables
      taskSpec:
        metadata: {}
        spec: null
        stepTemplate:
          name: ""
          computeResources: {}
          workingDir: /workspace/source
        steps:
        - image: golang:1.19
          name: hello-world
          computeResources: {}
          script: |
            #!/usr/bin/env sh
            echo "hello world"
        workspaces:
        - description: The workspace used to store the cloned git repository and the
            generated files
          mountPath: /workspace
          name: output
    - name: build-make-linux
      runAfter:
      - hello-world
      taskRef:
        params:
        - name: url
          value: https://github.com/jenkins-x/jx3-pipeline-catalog.git
        - name: revision
          value: versionStream
        - name: pathInRepo
          value: tasks/go/pullrequest/build-make-linux.yaml
        resolver: git
      workspaces:
      - name: output
        workspace: pipeline-ws
    - name: check-registry
      runAfter:
      - build-make-linux
      taskRef:
        params:
        - name: url
          value: https://github.com/jenkins-x/jx3-pipeline-catalog.git
        - name: revision
          value: versionStream
        - name: pathInRepo
          value: tasks/go/pullrequest/check-registry.yaml
        resolver: git
      workspaces:
      - name: output
        workspace: pipeline-ws
    - name: build-container-build
      runAfter:
      - check-registry
      taskRef:
        params:
        - name: url
          value: https://github.com/jenkins-x/jx3-pipeline-catalog.git
        - name: revision
          value: versionStream
        - name: pathInRepo
          value: tasks/go/pullrequest/build-container-build.yaml
        resolver: git
      workspaces:
      - name: output
        workspace: pipeline-ws
    - name: promote-jx-preview
      runAfter:
      - build-container-build
      taskRef:
        params:
        - name: url
          value: https://github.com/jenkins-x/jx3-pipeline-catalog.git
        - name: revision
          value: versionStream
        - name: pathInRepo
          value: tasks/go/pullrequest/promote-jx-preview.yaml
        resolver: git
      workspaces:
      - name: output
        workspace: pipeline-ws
    workspaces:
    - name: pipeline-ws
  taskRunTemplate:
    serviceAccountName: tekton-bot
  timeouts:
    pipeline: 1h0m0s
  workspaces:
  - name: pipeline-ws
    volumeClaimTemplate:
      metadata:
        creationTimestamp: null
      spec:
        accessModes:
        - ReadWriteOnce
        resources:
          requests:
            storage: 1Gi
      status: {}
